resources/magento2/validation.properties

```
@@ -10,3 +10,4 @@ validator.file.alreadyExists={0} already exists
 validator.file.cantBeCreated={0} can't be created
 validator.class.alreadyDeclared={0} already declared in the target module
 validator.magentoVersionInvalid=Please specify valid Magento version or use 'any' keyword as default
+validator.magentoNotExistTargetClass=Target class is not found. Check the di.xml file
```

As far as this validation message is related to class, please add it in `validator.class` scope
```suggestion
validator.class.targetClassNotFound=Target class is not found. Check the di.xml file
```

---

Hi @mmezhensky could you please adjust the name, and also adjust it where it is used?
Thank you.

---

done, please check. Tnx!